python-avocado.spec

```diff
@@ -151,6 +151,7 @@ popd
 
 %install
 %{__python} setup.py install --root %{buildroot} --skip-build
+%{__mv} %{buildroot}%{python_sitelib}/avocado/etc %{buildroot}
```

Comment:
I don't know whether there exist utility to follow the standard path, but usually the config files are only created when the file does not exists. If it exists `$name.rpmnew` is created instead and the original `/usr/share` is usually kept.

The least we should do is to `cp` and not `mv` and do it without only when the file is not already there.

Reply:
Oh, stupid me, this is only within the `buildroot` which then uses `install` (or config or whatever) to actually deploy it to the system-wide location, right? If so then I'd only `cp` it in order to keep the original location as well as the system-wide one, because that's what you get from `pip` installation, right? (I mean you won't get the `/etc` from pip, but if you create it it'll be respected)

Reply:
Yes, that's only for the RPM creation phase.  The resulting RPM file is a CPIO archive which gets unpacked into the file system.

I find that packaging should favor the target distro characteristics, and having duplicate configuration files inside the module doesn't look right to me.

In the end, packaging is exactly that, adjusting a general piece of software to the distribution way of doing things.